src/chrome/content/rules/Booklooker.de.xml

```diff
@@ -1,9 +1,20 @@
+<!--
+	Non-functional hosts
+		Different content:
+		- bjoerk.booklooker.de
+-->
 <ruleset name="booklooker.de">
-  <target host="booklooker.de" />
-  <target host="*.booklooker.de" />
+	<target host="booklooker.de" />
+	<target host="www.booklooker.de" />
+	<target host="api.booklooker.de" />
+	<target host="bms.booklooker.de" />
+	<target host="dev.booklooker.de" />
```

Comment:
This target has a different behaviour over HTTP and HTTPS. Plus, we don't have a working test url. Given we had issues recently with securing `dev` subdomains, I'd rather not add it.